### PR TITLE
skip digest for component constructors

### PR DESCRIPTION
**Pull request by mandelsoft**
#### What this PR does / why we need it
This PR adds the option `--skip-digest-generation` to the `ocm add components` command.
It generally disables the digest generation for all added resources.
Additionally, the resource description may set `skipDigestGeneration` to `true` to disable the digest 
generation  for this particular resource.

#### Which issue(s) this PR fixes

Fixes https://github.com/open-component-model/ocm-project/issues/322

### CODE REVIEW

File: docs/reference/ocm_add_componentversions.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ componentversions, componentversion, cv, components, component, comps, comp, c
       --lookup stringArray        repository name or spec for closure lookup fallback
   -O, --output string             output file for dry-run
   -R, --replace                   replace existing elements
-  -S, --scheme string             schema version (default "v2")
```
Comment (jakobmoellerdev) [inline docs/reference/ocm_add_componentversions.md:30]: Why was this removed in this PR? It looks like a different fix.

**Review by jakobmoellerdev — COMMENTED** [review]
Generally, LGTM but looks like another fix slipped into this one regarding the default scheme handling.

File: docs/reference/ocm_add_componentversions.md (hunk 1 above)
Comment (mandelsoft) [reply docs/reference/ocm_add_componentversions.md:30]: because the AddFlags was missing....

**Review by jakobmoellerdev — APPROVED** [review]

**Review by frewilhelm — APPROVED** [review]
